### PR TITLE
Fixed the datatype for min_price & max_price size 

### LINKED ISSUE #24428: Need to change `wc_product_meta_lookup` table structure datatype size for min_price and max_price
**Describe the bug**
If you set your product price greater than equal 100,000,000, you will face the issues that backend product list table price filter ( desc ) not working as well as woocommerce `widget_price_filter` also not working. After debugging I found that if you set any product price above or greater than above, woocommerce set value of `min_price/max_price` field with 99999999.99 ( maximum size and decimal point ) of table `wc_product_meta_lookup`.

**To Reproduce**
1. Go to backend Products and add / edit any product
2. Set its price to 100,000,000 to greater and save it.
3. Now filter products list table by price DESC.
4. Or add "Filter Products by Price" widget in shop sidebar and see the widget price range.

**Screenshots**
Table columns value
Product List Table filter
Price filter widget

**Expected behavior**
For widget "Filter Products by Price" the price range should be set with the maximum or minimum value of products.
For backend products list table product should be shown as price DESC value properly.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>

WordPress environment
--
WordPress address (URL): |   | http://localhost/wp
Site address (URL): |   | http://localhost/wp
WooCommerce version: |   | 3.7.0
WooCommerce REST API package: |   | 1.0.2 /data/html/wp/wp-content/plugins/woocommerce/packages/woocommerce-rest-api
Log directory writable: |   | /data/html/wp/wp-content/uploads/wc-logs/
WordPress version: |   | 5.2.2
WordPress multisite: |   | –
WordPress memory limit: |   | 256 MB
WordPress debug mode: |   |  
WordPress cron: |   |  
Language: |   | en_US
External object cache: |   | –

</details>

### PR DESCRIPTION
**Pull request by rohanvakharia**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #24428 .

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Tweak - Increased range and precision for min_price and max_price in the product meta lookup table.

### CODE REVIEW

**Comment by rohanvakharia** [comment]
I have assumed that this will go to milestone 3.10.0 and create the code.

If you are planning for different milestone let me know, I will update the code accordingly.

File: includes/wc-update-functions.php [hunk 1]
```diff
@@ -2046,3 +2046,19 @@ function wc_update_370_mro_std_currency() {
...
 	WC_Install::update_db_version( '3.7.0' );
 }
+
+/**
+ * Increase column size
+ */
+function wc_update_3100_increase_size_of_column() {
+	global $wpdb;
+	$wpdb->query( "ALTER TABLE {$wpdb->prefix}wc_product_meta_lookup MODIFY COLUMN `min_price` decimal(19,4) NULL default NULL" );
+}
```
Comment (peterfabian) [inline includes/wc-update-functions.php]: I think the `max_price` column should be updated as well here, no?

**Review by peterfabian — CHANGES_REQUESTED** [review]
Thanks for your contribution. There will be no 3.10, so we could try it for 4.0, but beta is about to be tagged soon.

As [@vedanshujain mentioned](https://github.com/woocommerce/woocommerce/issues/24428#issuecomment-537401291), we cast the numbers to `decimal(10 ,2)` in some other places in the code, so I think we need to reflect the change elsewhere as well, e.g. in
`wc_update_product_lookup_tables_column` function.

File: includes/wc-update-functions.php (hunk 1 above)
Comment (rohanvakharia) [reply includes/wc-update-functions.php]: Yes. I have updated the code.

**Comment by rohanvakharia** [comment]
@peterfabian Should I update the code for version 4.0?

**Comment by peterfabian** [comment]
@rohanvakharia Yes, please. Plus there are some conflicts that need to be resolved.

**Comment by rohanvakharia** [comment]
@vedanshujain @peterfabian Code is updated for 4.0 and conflicts are resolved, so you can proceed with code review.

**Review by peterfabian — APPROVED** [review]

**Review by peterfabian — APPROVED** [review]
Works fine as far as I can see. Thanks!

**Pull request merged**